app/presenters/publishing_api/default_news_image_helper.rb

```diff
@@ -0,0 +1,16 @@
+module PublishingApi::DefaultNewsImageHelper
```

Comment:
We normally put presenter helpers in the `app/helpers` directory.  I think the only other helper here (`UpdateTypeHelper`) probably ought to be moved to.

Reply:
All good, I'll swing back to that. 👍